### ISSUE TITLE
🐛 BUG:  Need to restart server to update the tailwind classes

### ISSUE THREAD

**Issue by imanaspaul** [snowpackjs/astro#797]
### What package manager are you using?

npm

### What operating system are you using?

Linux

### Describe the Bug

Each time I have to restart the server `npm start` to update the classes for tailwindcss. 

### Steps to Reproduce

1. `<h1 class="text-lg">Hello astro</h1>`
2. Now if you refresh the webpage it won't take any affect.
3. You have to re run the server
4. `npm start`

**Comment by akant-poznan** [comment]
I can reproduce described behaviour when there is `scss` involved (`astro@0.20.7`). 

This setup works as expected with `../styles/global.css` file:
```html
<link rel="stylesheet" href={Astro.resolve('../styles/global.css')}>
```

The same requires manual restart with `../styles/global.scss` file.

**Comment by Gee-Bee** [comment]
I can reproduce described behaviour when there is `scss` involved (`astro@0.20.7`). 

This setup works as expected with `../styles/global.css` file:
```html
<link rel="stylesheet" href={Astro.resolve('../styles/global.css')}>
```

The same requires manual restart with `../styles/global.scss` file.

**Comment by FredKSchott** [comment]
Thanks for the report! We believe that this has been fixed in astro v0.21, going out today. Let us know if you can still reproduce the issue after updating to v0.21 and we'll be happy to take a look.
